tests/test_sanity.py

```diff
@@ -67,6 +68,19 @@ def test_rbspy(
         assert_collapsed(process_collapsed, check_comm=True)
 
 
+@pytest.mark.parametrize("runtime", ["nodejs"])
+def test_nodejs(
+    tmp_path: Path,
+    application_pid: int,
+    assert_collapsed,
+    gprofiler_docker_image: Image,
+    runtime: str,
+) -> None:
+    with SystemProfiler(1000, 3, Event(), str(tmp_path), "fp", True, 0) as profiler:
```

Comment:
I genuinely don't like the invocation here, too many const arguments. kwargs will look better

Reply:
k